scripts/install-hooks.bash

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
How does this work? Should we run `install-hooks` manually once or the pre-commit hook gets installed automatically? 🤔

Reply:
Currently we should run hooks file once manually on root folder.